guides/20240823_how_to_setup_postgresql_playground_in_daytona.md

```diff
@@ -0,0 +1,372 @@
+
+  ```
+  postgresql-playground-in-daytona/
+  ├── .devcontainer/
+  │   ├── .env
+  │   ├── docker-compose.yml
+  │   └── devcontainer.json
+  ```
+
+- ### **Step 6:** Go back to the top level
+
+  Paste the code below to go back to the top level of the directory you created
+
+  ```bash
+  cd ..
+  ls -al
+  ```
+
+  and you should see this output in the terminal:
+  
+  ```
+  ├── .
+  ├── ..
+  ├── .devcontainer
+  ├── .git
+  ```
+
+- ### **Step 7:** Initialize and make commit
+
+  Paste the code below to initialize git and commit the changes you made to your directory.
+
+  ```bash
+  git init
+  git add .
+  git commit -m "inital commit"
+  ```
+
+- ### **Step 8:** Create a repository in GitHub
+
+  Create a repository without README, LICENSE or .gitignore files from GitHub web using the name of the directory you created. Mine is `postgresql-playground-in-daytona`.
+
+  You should see a code block similar to this from your GitHub web. Copy it and paste in your terminal or Windows Powershell for Windows PC users(Git must be installed in it)
+
+  ```bash
+  git remote add origin https://github.com/YOUR-GITHUB-USERNAME/YOUR-DIRECTORY-NAME.git
+  git branch -M main
+  git push -u origin main
+  ```
+
+  After you run the code, you'll be prompted to input your GitHub username and password.
+
+  You can find the GitHub repository where my devcontainer config is located which I used for this guide [here](https://github.com/c0d33ngr/postgresql-playground-in-daytona). I later added a README and LICENSE files which wasn't neccessary to follow along with this guide.
+
+Now, we have successfully created a GitHub repository needed to spin up a devcontainer for PostgreSQL using Daytona.
+
+## Setting Up PostgreSQL in Daytona
+
+Before starting this section be sure `daytona` is installed in your PC
+
+- ### Step 1
+
+  Run the code below to setup `daytona` server
+
+  ```bash
+  daytona server
+  ```
+
+  Your output should be similar to the screenshot below
+
+  ![screenshot of running daytona server](assets/20240823_how_to_setup_postgresql_playground_in_daytona_img_1.jpg)
+
+  Choose yes and should see similar output in the screenshot below
+
+  ![screenshot of successfully running daytona](assets/20240823_how_to_setup_postgresql_playground_in_daytona_img_2.jpg)
+  
+- ### Step 2
+
+  Run the command below to add your git provider if you haven't setup one
+
+  ```bash
+  daytona git provider add
+  ```
+
+  Follow the prompts after running the command to setup your provider. In our case it's GitHub
+
+- ### Step 3
+
+  Run this command in terminal to add your preferred IDE.
+
+  ```bash
+  daytona ide
+  ```
+
+- ### Step 4
+
+  Modified the terminal command below to create the dev environment of the repository you created in GitHub and follow the prompts after you run it. Don't forget to use the correct GitHub URL, in my case it's `https://github.com/c0d33ngr/postgresql-playground-in-daytona.git`
```

Comment:
for me this fails 
![image](https://github.com/user-attachments/assets/6f527ae1-3b98-4c93-8b8b-e95167aeba77)

Reply:
It was because I didn't include "image" value. I had to use base ubuntu from microsoft in the modified code with screen recording attached to the PR